docs/config/README.md

```diff
@@ -412,7 +412,15 @@ By default it only changes color. If you also want to change its shape take a
 look at [this example](#with-custom-error-shape).
 
 ::: warning
+
 `error_symbol` is not supported on elvish shell.
+
+:::
+
+::: warning
+
+`vicmd_symbol` is not supported for Bash's vi mode.
```

Comment:
I think it might be better to just list the shells that are supported. For instance the PowerShell vi mode isn't supported either.

Reply:
TIL Powershell has vi mode